enos/modules/alb_boundary/main.tf

```diff
@@ -1,5 +1,4 @@
 resource "aws_security_group" "boundary_ingress_sg" {
-  name   = "${var.lb_name_suffix}-server-lb"
```

Comment:
Names like these are not required and the provider will assign random, unique names for them. Other values here had sane defaults and their declaration can be removed from our Terraform module.